plugins/native/configs/config.d/10_native.json

```diff
@@ -22,5 +22,19 @@
   {
     "comp": "-2",
     "translucency-support": true
+  },
+  ".*(splat)":
+  {
+    "point-sprites": true,
+    "point-size": 1,
+    "point-type": "gaussian",
+    "scalars": "color",
+    "comp": "-2",
+    "bg-color": "0, 0, 0",
+    "up": "-Y",
+    "trackball": true,
+    "camera-position": "0,1,-5.2",
```

Comment:
camera-position is unexepected in config file. Why not camera-direction ?

Reply:
There is `camera-position` and `camera-focal-point`. No reason to give the direction.  
I know it's a bit unexpected to have the camera direction here, but that's necessary.

Reply:
That means the data bounds of any splat file are always the same ?

Reply:
Yeah

Reply:
im not sure to follow the logic, lets discuss.